src/ext/java/org/opentripplanner/ext/stopconsolidation/DecorateConsolidatedStopNames.java

```diff
@@ -34,18 +34,16 @@ private void replacePrimaryNamesWithSecondary(Itinerary i) {
     i.transformTransitLegs(leg -> {
       if (leg instanceof ScheduledTransitLeg stl && needsToRenameStops(stl)) {
         var agency = leg.getAgency();
-        return new ConsolidatedStopLeg(
-          stl,
-          service.agencySpecificName(stl.getFrom().stop, agency),
-          service.agencySpecificName(stl.getTo().stop, agency)
-        );
+        var from = service.primaryStop(stl.getFrom().stop.getId());
+        var to = service.agencySpecificStop(stl.getTo().stop, agency);
```

Comment:
Add a comment explaining the use of primary stop in one instance vs. the use of secondary stop in the other.

Reply:
I addressed it in 7692ef10fa9d8a5d51b371c7585ecbb9df4a4f2d